packages/core/src/interfaces/components.ts

```diff
@@ -121,6 +121,10 @@ export interface ZoomBarsOptions {
 	 * currently only the top position is supported
 	 */
 	top?: ZoomBarOptions;
+	/**
+	 * options related to tool bar above zoom bar
+	 */
+	toolBar?: ToolBarOptions;
```

Comment:
this should be a root-level config in charting options

Reply:
Updated in https://github.com/carbon-design-system/carbon-charts/pull/717/commits/50a9137867775ff491405b4b22971d9968682fa8.